File: 프로그래머스/unrated/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.py
def solution(arr):
    answer = []
    for n in arr:
        if (n>=50) & (n%2==0):
            answer.append(int(n/2))
        elif (n<50) & (n%2!=0):
            answer.append(int(n*2))
        else:
            answer.append(n)
    return answer

File: 프로그래머스/unrated/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.swift
import Foundation

func solution(_ arr:[Int]) -> [Int] {
    var temp:[Int] = []
    for n in arr {
        if (n<50 && n%2 != 0) {
            temp.append(n*2)
        } else if (n>=50 && n%2 == 0) {
            temp.append(n/2)
        } else {
            temp.append(n)
        }
    }
    return temp
}